ann_benchmarks/runner.py

```diff
@@ -223,6 +232,8 @@ def run_docker(definition, dataset, count, runs, timeout, batch, cpu_limit,
         cpu_limit = "0-%d" % (multiprocessing.cpu_count() - 1)
     print('Running on CPUs:', cpu_limit)
 
+    logic_cpu_num = psutil.cpu_count(logical=True)
```

Comment:
We've been running the benchmark in a single-cpu mode for quite some time. The argument is that most algorithms are single-threaded, but can be trivially parallelizeable by just using multiple threads for the queries anyway, so it doesn't really affect the throughput per CPU anyway.